dotcom-rendering/cypress/integration/parallel-4/signedin.spec.js

```diff
@@ -40,6 +40,17 @@ describe('Signed in readers', function () {
 
 	it('should not display signed in texts when users are not signed in', function () {
 		cy.visit(`Article?url=${articleUrl}`);
+		cy.scrollTo('bottom', { duration: 300 });
+		// We need this second call to fix flakiness where content loads in pushing the page
+		// down and preventing the scroll request to actually reach the bottom. We will fix
+		// this later when we've defined fixed heights for these containers, preventing CLS
+		cy.scrollTo('bottom', { duration: 300 });
+		// Wait for hydration
+		cy.get('gu-island[name=DiscussionContainer]').should(
```

Comment:
Why do we check hydration in this test?

Reply:
Because when I say 'hydration' here what I really mean is wait for the Island to load the content onto the page. So it's not actually SSR + hydration, it's a pure client side load. I'll update the comment to better reflect this

Reply:
Aside, maybe `data-gu-hydrated` should be `data-gu-ready`?

Reply:
This change to `data-gu-ready` has now been done in #3872